Add 3 kb articles (#790)
telerik/maui-docs: knowledge-base/set-cultureinfo-on-app-level-maui.md

---

> title: Setting the Culture Info on App Level

Reviewer: ```suggestion
title: Setting the Culture Info on the App Level
```

> description: How to set the culture info on app level different than the device culture.

Reviewer: ```suggestion
description: How to override the device culture settings and apply your preferred localization settings within your Telerik UI for .NET MAUI app.
```

type: how-to

> page_title: Setting the Culture Info on App Level

Reviewer: ```suggestion
page_title: Setting the Culture Info on the App Level
```

slug: set-cultureinfo-on-app-level-maui
position: 
tags: maui, culture info, application culture, device culture, dotnet, maui
res_type: kb
---

| Version | Product | Author | 
| --- | --- | ---- | 
| 6.7.0 | Telerik UI for .NET MAUI | [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova)| 

## Description

> This article shows you how to apply current culture (localization) on application level (app localization). For example if the device culture is set to Spanish, or German, or French, and you want to have different app culture (localization) than the device culture, you can explicitly set culture info on app level.

Reviewer: You can override the device culture settings by explicitly setting the culture info on the app level. For example, even though the device regional and language preferences might be set to Spanish, you can choose to always apply German localization to your app instead.

## Solution

The solution applies for each platform inside the Application/Platforms folder:

### Android

> On Android&mdash;Inside the MainActivity class, implement `SetLocale()` method to change the culture info:

Reviewer: ```suggestion
On Android&mdash;Inside the `MainActivity` class, implement a `SetLocale()` method to change the culture info:
```
